modules/internal/localeModels/numa/LocaleModel.chpl

```diff
@@ -502,40 +496,16 @@ module LocaleModel {
     //
     const dnode =  chpl_nodeFromLocaleID(loc);
     const dsubloc =  chpl_sublocFromLocaleID(loc);
```

Comment:
Just thinking out loud: I wonder if it'd be faster to have a method that can set the locale and sublocale instead of having to make two calls? Oh wait, I keep confusing whether `chpl_sublocFromLocaleID()` or `chpl_task_setSubloc()` is the expensive call, so `chpl_sublocFromLocaleID()` is probably fast enough that it doesn't matter